pkg/odo/cli/config/set.go

```diff
@@ -49,6 +51,13 @@ func NewSetOptions() *SetOptions {
 func (o *SetOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
 	o.paramName = args[0]
 	o.paramValue = args[1]
+
+	if len(o.componentContext) > 0 {
```

Comment:
Eventually this function should be located in the utils go file and applied as a validation check against `--context` as a global flag.

But that change will come in the future.

Reply:
resolved